edx_proctoring/templates/proctoring/seq_proctored_exam_ready_to_start.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n %}
+<div class="sequence proctored-exam success-top-bar instructions" data-exam-id="{{exam_id}}" data-exam-started-poll-url="{{exam_started_poll_url}}">
+  <h3>
+    {% blocktrans %}
+      Your Proctoring Installation and Set Up is Complete
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Placeholder content. <strong>Please keep the proctoring software open and running while you complete your exam</strong>.
+    {% endblocktrans %}
+  </p>
+  <div class="proctored-exam-message">
+    <h3>
+    {% blocktrans %}
+      You May Begin Your Exam Now
+    {% endblocktrans %}
+    </h3>
+    <p>
+      {% blocktrans %}
+        Your proctored and timed exam is now ready. Once you start, <strong>you will have {{total_time}} to
```

Comment:
When you start your exam you will have ```{{total_time}}``` to complete it. You cannot stop the timer once you start. 

If the allotted time expires before you mark your exam as ended, the answers you have completed up to that point are submitted for grading, and your proctoring session is uploaded for review.